.travis.yml

```diff
@@ -1,24 +1,46 @@
+dist: trusty
 language: python
+# qiita travis has set sudo to false
 sudo: required
 group: deprecated-2017Q4
 env:
   global:
     - PYTHON_VERSION=3.5
 before_install:
+  - sudo apt-get install libevent-dev
   - redis-server --port 7777 &
   - wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   - chmod +x miniconda.sh
   - ./miniconda.sh -b
+  - rm miniconda.sh
+  - df -h
   - export PATH=/home/travis/miniconda3/bin:$PATH
   # Update conda itself
   - conda update --yes conda
 install:
+  # Hard limit for transaction log, to prevent VHD from filling during tests
+  - psql -c "ALTER SYSTEM SET max_wal_size=2"
+  - psql -c "ALTER SYSTEM SET wal_keep_segments=0"
+  - psql -c "SELECT pg_reload_conf()"
+  # Verify log directory exists, postgresql processes exist, and postgresql is
+  # listening on port
+  - sudo find /var/lib/postgresql | grep -i log
+  - ps -A | grep post
+  - netstat -tulpn
   # Install Qiita first since it is needed for the environment and the tests
-  - conda create --yes -n qiita python=2.7 pip nose flake8 pyzmq networkx
-    pyparsing natsort mock future libgfortran 'pandas>=0.18' 'scipy>0.13.0'
-    'numpy>=1.7' 'h5py>=2.3.1'
+  - travis_retry conda create -q --yes -n qiita python=2.7 pip nose flake8
+    pyzmq 'networkx<2.0' pyparsing natsort mock future libgfortran seaborn nltk
+    'pandas>=0.18' 'matplotlib>=1.1.0' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
   - source activate qiita
-  - pip install sphinx==1.5.5 sphinx-bootstrap-theme ipython[all]==2.4.1
+  - pip install pip==18.1
```

Comment:
The conda command in lin 31 is installing pip, can you remove this line and specify `pip==18.1` in line 31?